### ISSUE TITLE
llama3.1 deployment example notebook appears to use the wrong # of L4 GPUs for g2-standard-8 machine

### ISSUE THREAD

**Issue by amygdala** [GoogleCloudPlatform/vertex-ai-samples#3372]
For this notebook example: https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/model_garden/model_garden_pytorch_llama3_1_deployment.ipynb

...I believe that for nvidia L4s, for a g2-standard-8 machine, only one GPU can be used. 
See the g2 section of https://cloud.google.com/compute/docs/gpus

cc @kathyyu-google (as it looks like you touched this notebook recently)

## Expected Behavior

the example works :)

## Actual Behavior

When running `deploy_model_vllm` with the  config as specified in this cell:
```
if "70" in base_model_name:
    accelerator_type = "NVIDIA_L4"
    machine_type = "g2-standard-8"
    accelerator_count = 8
...
else:
    raise ValueError(
        f"Recommended GPU setting not found for: {accelerator_type} and {base_model_name}."
    )
```
…the error is:

`InvalidArgument: 400 The accelerator count "8" is not allowed. The allowed numbers of accelerator "NVIDIA_L4" to use with machine type "g2-standard-8" are: [1].`

## Steps to Reproduce the Problem

Run the notebook

## Specifications

- Version:
- Platform:

**Comment by xiangxu-google** [comment]
Fixing in https://github.com/GoogleCloudPlatform/vertex-ai-samples/pull/3373

**Comment by kathyyu-google** [comment]
Marking this issue as closed because #3373 was merged, changing the machine type to `g2-standard-96`. Please reopen if needed, thanks!
